### PR TITLE
chore: migrate to owl bot

### PR DESCRIPTION
**Pull request by parthea**
This PR migrates from autosynth to [owl bot](https://github.com/googleapis/repo-automation-bots/tree/master/packages/owl-bot).  PRs will be automatically opened when there are template updates from [synthtool](https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/python_library). With owl bot, on every PR, a post-processor image will run so that we won't have to ask contributors to run [blacken](https://github.com/googleapis/python-bigquery-sqlalchemy/blob/master/synth.py#L76). Googlers see [go/owlbot-user-guide](https://goto.corp.google.com/owlbot-user-guide).

### CODE REVIEW

File: owlbot.py [hunk 1]
```diff
@@ -27,52 +27,19 @@
...
-[sqla_testing]
-requirement_cls=pybigquery.requirements:Requirements
-profile_file=.profiles.txt
-
-[db]
-default=bigquery://
-bigquery=bigquery://
-"""
-)
-
```
Comment (tseaver) [inline owlbot.py:71]: Is this change correct? Dropping the replacements and excluding `setup.cfg` and `noxfile.py` from synth makes them susceptible to drift.

**Comment by parthea** [comment]
Blocked by #139

File: owlbot.py (hunk 1 above)
Comment (parthea) [reply owlbot.py]: Done.

**Comment by parthea** [comment]
Build failure is due to #187

**Comment by parthea** [comment]
Blocked by #187

**Comment by jimfulton** [comment]
@googlebot I consent.

**Comment by parthea** [comment]
Thanks @jimfulton!

**Comment by parthea** [comment]
@tseaver This is ready for review

**Review by jimfulton — APPROVED** [review]

**Pull request merged**